### PR TITLE
Created models for users, dreams, and bookings

### PR DESCRIPTION
**Pull request by dimitripantelli**
On the command line, hence extraneous test files made

### CODE REVIEW

**Review by SofieFiskerstrand — COMMENTED** [review]
Looks good :)

File: app/models/dream.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Dream < ApplicationRecord
+  belongs_to :user
```
Comment (alisonclare93) [inline app/models/dream.rb:2]: needs a has_many :bookings 
and has_many users through bookings
(Deji popped by just as we were looking at it 👍 )

**Review by alisonclare93 — COMMENTED** [review]
Comment on Dream model - associations

File: app/models/dream.rb (hunk 1 above)
Comment (dimitripantelli) [reply app/models/dream.rb:2]: Added! Thanks

**Review by alisonclare93 — APPROVED** [review]
Looks good!

**Pull request merged**